greymatter/grocerylist/apple.cue

```diff
@@ -60,29 +53,41 @@ Apple: {
 			domain_key: "edge"
 			route_key:  Name
 			route_match: {
-				path: "/services/apple/"
+				path: "/services/grocerylist/apple/"
 			}
 			redirects: [
 				{
-					from:          "^/services/apple$"
+					from:          "^/services/grocerylist/apple$"
 					to:            route_match.path
 					redirect_type: "permanent"
 				},
 			]
 			prefix_rewrite: "/"
 		},
 
+		// Grey Matter catalog service definition for apple
 		#catalog_entry & {
 			name:                      Name
-			mesh_id:                   mesh.metadata.name
+			mesh_id:                   mesh.name
 			service_id:                Name
 			version:                   "v1.0.0"
-			description:               "Apple service that serves up apples"
-			api_endpoint:              "/services/\(Name)/"
-			api_spec_endpoint:         "/services/\(Name)/"
+			description:               "EDIT ME: service description goes here"
+			api_endpoint:              "/services/grocerylist/\(Name)/"
```

Comment:
If our pattern for routes will always be /services perhaps we should also adopt the following:

/services/\(AppPath)/\(Name)/

This would cut down on cutting and pasting "grocerylist" everywhere.

I might also advocate to add AppPath to inputs.cue.

Reply:
It's a good idea. We can set the value of AppName to the name of the project provided to init `greymatter init myproject`. Then the users can change it if they want to but at least it'd have a default value.

Reply:
This has already been adopted in init. This is the resulting auto generated string